postprocessing/processors/CreateReductionScript_Processor.py

```diff
@@ -0,0 +1,53 @@
+from .base_processor import BaseProcessor
+from postprocessing import reduction_script_writer
+
+
+import logging
+import sys
+
+
+class CreateReductionScript_Processor(BaseProcessor):
```

Comment:
remove the _ from the class name and rename the file `create_reduction_script_processor.py`